src/app/@theme/components/language-switcher/language-switcher-list/language-switcher-list.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, Input } from '@angular/core';
+import { NbPopoverDirective, NbLayoutDirection, NbLayoutDirectionService } from '@nebular/theme';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+  selector: 'ngx-lang-switcher-list',
+  templateUrl: './language-switcher-list.component.html',
+  styleUrls: ['./language-switcher-list.component.scss'],
+})
+
+export class LangSwitcherListComponent {
+
```

Comment:
Not sure, but I think in most places in the code, there is no empty line at the start of classes

Reply:
let's at least remove the empty line right before the class declaration